README.fr-fr.md

```diff
@@ -0,0 +1,227 @@
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>TW/HK/MO</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>Japon</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>Corée</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td rowspan="2">Genshin Impact</td>
+      <td>Global</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>Chine continentale</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td rowspan="2">Honkai: Star Rail</td>
+      <td>Global</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>Chine continentale</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>:white_check_mark:</td>
+      <td>N/A</td>
+    </tr>
+  </tbody>
+</table>
+
+> **Note** :
+> Le tableau ci-dessus sert à illustrer les fonctionnalités que Collapse prend actuellement en charge. Cette liste sera continuellement mise à jour au fur et à mesure que de nouvelles fonctionnalités seront ajoutées à Collapse et que de nouveaux jeux sortiront. Si vous avez des problèmes avec l'une des fonctionnalités supportées ci-dessus, vérifiez notre onglet GitHub Issues, si aucun problème actif n'est trouvé pour cette fonctionnalité, créez une Issue.
+> Veuillez garder à l'esprit que la fonction de conversion de jeu n'est actuellement disponible que pour Honkai Impact 3rd. Les autres jeux miHoYo/Cognosphere Pte. Ltd. ne sont pas prévus pour l'instant.
+
+# Télécharger les versions prêtes à l'emploi
```

Comment:
As this section will be updated quite a lot during the release. It's better to just simplify it by redirecting the user to the `releases` section so they can always download the latest version without the need for us to update this section every time there is an update.

Releases link : https://github.com/neon-nyan/CollapseLauncher/releases

Reply:
I think the review selection is incorrect? Both comments seem relevant to me, do you just want to replace "# Télécharger les versions prêtes à l'emploi" and the various points that make it up?

Reply:
yeah, basically just delete these lines
```md
[<img src="https://user-images.githubusercontent.com/30566970/172445052-b0e62327-1d2e-4663-bc0f-af50c7f23615.svg" width="320"/>](https://github.com/neon-nyan/Collapse/releases/download/CL-v1.71.13/CL-1.71.13_Installer.exe)
> **Note** : La version pour ce build est `1.71.13` (Sortie le : 2 Octobre 2023).
[<img src="https://user-images.githubusercontent.com/30566970/172445153-d098de0d-1236-4124-8e13-05000b374eb6.svg" width="320"/>](https://github.com/CollapseLauncher/Collapse/releases/download/v1.72.4-pre/CL-1.72.4-preview_Installer.exe)
> **Note** : La version pour ce build est `1.72.4` (Sortie le : 9 Octobre 2023).
```
so it only contains reference that tells people to go to Releases section

Sorry about the double comment, I forgot to set my earlier comment to `Request Changes`